### ISSUE TITLE
"ID_Einst_Hysterese_HR_verkuerzt_akt"

### ISSUE THREAD

**Issue by colognebroker** [Bouni/luxtronik#54]
Is this the Hyterese for dometic water ?
It would be nice to have it and to change it !

**Comment by Bouni** [comment]
I can't say that. Ideally you figure that out yourself and let us know so that we can add this as a known parameter.

1. make sure you installed luxtronik -> `pip install luxtronik`
2. create a script `lux.py` with this content (adjust the ip):

```
from luxtronik import Luxtronik

l = Luxtronik('192.168.178.100', 8889)

p = l.parameters.get("ID_Einst_Hysterese_HR_verkuerzt_akt")

print(p)
```

3. Not the value the script spits out
4. Change the hysteresis via the heatpumps web UI
5. Run the script again to see if it changed to what you've just set via th UI

Let us know if thats the right parameter

**Comment by Bouni** [comment]
Another way is this: go to heatpump24.com and log in, there you see the parameters:

![grafik](https://user-images.githubusercontent.com/948965/207912561-b0984ef6-d737-40b0-a659-c5d0b4476573.png)

The number in front of the parameter matches the number in the python-luxtronik module:

https://github.com/Bouni/python-luxtronik/blob/850daa40041a00bbd0d69f3a98845ae29e09f493/luxtronik/parameters.py#L99

So the parameter you're looking for is mos likely `ID_Einst_BWS_Hyst_akt`

**Comment by BenPru** [comment]
Yes it is `parameters.ID_Einst_HRHyst_akt`. In https://github.com/BenPru/luxtronik you have a writeable number `number.luxtronik2_heating_hysteresis`.
